package.json

```diff
@@ -29,5 +28,8 @@
     "client",
     "example",
     "examples/browser"
-  ]
-}
\ No newline at end of file
+  ],
+  "dependencies": {
+    "@types/vscode": "1.33.0"
```

Comment:
Can this be a devDependency?

Reply:
yeah just tested it with it as a dev dependency and it works so added that